complete/build.gradle

```diff
@@ -1,34 +1,34 @@
 plugins {
-	id 'org.springframework.boot' version '2.3.2.RELEASE'
-	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+	id 'org.springframework.boot' version '2.4.1'
+	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+	id 'org.asciidoctor.convert' version '1.5.8'
 	id 'java'
-	id 'org.asciidoctor.convert' version '1.5.6'
 }
 
 group = 'com.example'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '1.8'
 
-asciidoctor {
-	sourceDir 'src/main/asciidoc'
```

Comment:
I reinclude this line in one the next commits (9af112a87359a37a175124bedd4a8cfe4512b5e3)